frontend/packages/dsco/src/divider/dividerContentfulDefinition.ts

```diff
@@ -0,0 +1,15 @@
+// Creates a definition for the Divider component to be used in Contentful Studio
+import {ComponentDefinition} from '@contentful/experiences-sdk-react';
+
+export const DividerContentfulComponentDefinition: ComponentDefinition = {
+  id: 'divider',
+  name: 'Divider',
+  category: 'Custom Components',
+  variables: {
+    color: {
```

Comment:
misc: Excited to see how it transpiled by Contentfull. Is it able to fetch the possible prop values from the prop types?

Reply:
It should be able to, but I'm going to do that in a followup PR!

Reply:
I actually figured this out, and will add it here! It's pretty straightforward and we can keep this all together.